ui/src/test/java/edu/wpi/grip/ui/codegeneration/tools/PipelineGenerator.java

```diff
@@ -60,15 +61,35 @@ public Step addStep(OperationMetaData data) {
   }
 
   public void export(String fileName) {
-    
-    new Exporter(pipeline.getSteps(), Language.JAVA, 
-        codeDir.toPath().resolve(fileName + ".java").toFile(),
+    new Exporter(pipeline.getSteps(),
+        CodeGenerationOptions.builder()
+            .language(Language.JAVA)
+            .className(fileName)
+            .saveDir(codeDir.getAbsolutePath())
+            .implementVisionPipeline(false)
+            .packageName("")
+            .moduleName("")
+            .build(),
         true).run();
-    new Exporter(pipeline.getSteps(), Language.PYTHON, 
-        codeDir.toPath().resolve(fileName + ".py").toFile(),
+    new Exporter(pipeline.getSteps(),
+        CodeGenerationOptions.builder()
+            .language(Language.PYTHON)
+            .className(fileName)
+            .saveDir(codeDir.getAbsolutePath())
+            .implementVisionPipeline(false)
+            .packageName("")
+            .moduleName(fileName)
+            .build(),
         true).run();
-    new Exporter(pipeline.getSteps(), Language.CPP, 
-        codeDir.toPath().resolve(fileName + ".cpp").toFile(),
+    new Exporter(pipeline.getSteps(),
+        CodeGenerationOptions.builder()
+            .language(Language.CPP)
+            .className(fileName)
+            .saveDir(codeDir.getAbsolutePath())
+            .implementVisionPipeline(false)
+            .packageName("")
+            .moduleName("")
+            .build(),
```

Comment:
I like this better.
Can't velocity handle the `CodeGenerationOptions` being a java bean or does it really need to implement `Map`?